FeatureAreas.props

```diff
@@ -62,6 +62,7 @@
     <FeatureSplitViewEnabled>false</FeatureSplitViewEnabled>
     <FeatureProgressBarEnabled>false</FeatureProgressBarEnabled>
     <FeatureProgressRingEnabled>false</FeatureProgressRingEnabled>
+    <FeatureRadialGradientBrushEnabled>true</FeatureRadialGradientBrushEnabled>
```

Comment:
I believe this should be false, otherwise this feature will always build when using the inner loop solution file which I dont think is needed.

Reply:
I think this was set by this script line - should the script maybe be updated too?

https://github.com/microsoft/microsoft-ui-xaml/blob/1f48f11b3cd895b10206f9abc93defb8eeff26b5/tools/GenerateNewControlProject.ps1#L65

Reply:
yes. can you please update the script ?

Reply:
That will require adding a new case to the script logic since it sets multiple nodes that should have different values - I could try to tackle that in a separate PR

Reply:
Sure.

Reply:
Opened issue #1836 to track some potential fixes/improvements

Reply:
Thanks. Can you please update this instance manually and change it to false ?

Reply:
It's `false` for the InnerLoop project and `true` for the rest

Reply:
Ah. this is confusing. There is a set of flags in featureAreas.props (first half) which are defaults. It should be false there. I think the only ones that need to be true in that group are areas which are absolutely required to be able to build anything. This needs to change to false in your case.

The second half of featurearea.props is for the muxcontrols.sln build. It needs to be true there since that is what we build in the lab to test and release.

The Innerloop.props can have false or true. That is up to you.

Reply:
okay, I think that already matches the current state.